src/main/resources/l10n/JabRef_da.properties

```diff
@@ -825,15 +825,9 @@ web\ link=link
 
 What\ do\ you\ want\ to\ do?=Hvad vil du gøre?
 
-Will\ write\ XMP\ metadata\ to\ the\ PDFs\ linked\ from\ selected\ entries.=Skriver XMP metadata til PDF-filerne linket fra de valgte poster.
 
-Write\ BibTeXEntry\ as\ XMP\ metadata\ to\ PDF.=Skriv BibTeX-posten som XMP metadata til PDF.
 
 Write\ XMP=Skriv XMP
-Write\ XMP\ metadata=Skriv XMP metadata
```

Comment:
Did something go wrong at crowdin sync - or why was the existing translation removed?

Reply:
From what I see the key got removed or replaced without the Metadata

Reply:
I also meant the other deleted ones. On the one hand, in the other translations, they are just changed or simply kept. Therefore, I worry a bit. On the other hand, the affected languages seem ones that we might (temporarily?) remove.

<blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/JabRef?src=hash&amp;ref_src=twsrc%5Etfw">#JabRef</a> needs your help in <a href="https://twitter.com/hashtag/translations?src=hash&amp;ref_src=twsrc%5Etfw">#translations</a> --&gt; <a href="https://t.co/H8Pbz40QWZ">https://t.co/H8Pbz40QWZ</a> <a href="https://t.co/0p3voAdmkk"><img src="https://pbs.twimg.com/media/ERQkRQmXUAI3w2E?format=jpg&name=small"></a></p>&mdash; JabRef (@JabRef_org) <a href="https://twitter.com/JabRef_org/status/1230648669738803201?ref_src=twsrc%5Etfw">February 21, 2020</a></blockquote>